test/selectors/columnWidths-test.js

```diff
@@ -131,11 +131,11 @@ describe('columnWidths', function () {
 
     assert.deepEqual(
       columnGroupProps.map((column) => column.width),
-      [110, 70, 100, 100]
+      [110, 100, 70, 100]
```

Comment:
Here and below.
The values are different because `columnGroupProps` and `columnProps` are now sorted based on cell group type.